conjure-api/src/main/conjure/conjure-api.yml

```diff
@@ -57,6 +57,11 @@ types:
           - CUSTOM_SERVER
       ErrorDefinition:
         fields:
+          # TODO(pm): Should there be a message here? The docs aren't the same thing: devs can add more context to them,
```

Comment:
Conjure errors don't provide messages by design, the name is meant to be a key such that a FE component can build a human-readable message. Providing a user-facing message would require our servers to implement internationalization/translation, which is something our current architecture is designed to handle exclusively on the frontend.
While the server can describe what failed, the FE is better positioned to contextualize the failure in order to provide good user experience.

Reply:
Hadn't thought about i18n. Makes sense.